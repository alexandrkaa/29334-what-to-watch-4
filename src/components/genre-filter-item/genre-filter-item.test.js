import React from 'react';
import renderer from 'react-test-renderer';
import GenreFilterItem from './genre-filter-item.jsx';

it(`<GenreFilterItem /> should match snapshot`, () => {
  const mockGenre = `Thriller`;
  const onActiveItemChange = jest.fn();
  const tree = renderer
    .create(
        <GenreFilterItem
          movieGenre={mockGenre}
          isActive={true}
          onActiveItemChange={onActiveItemChange}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
