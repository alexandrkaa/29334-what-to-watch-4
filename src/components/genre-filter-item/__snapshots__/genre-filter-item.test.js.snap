// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GenreFilterItem /> should match snapshot 1`] = `
<li
  className="catalog__genres-item catalog__genres-item--active"
>
  <a
    className="catalog__genres-link"
    href="#"
    onClick={[Function]}
  >
    Thriller
  </a>
</li>
`;
